templates/specialMissions.php

```diff
@@ -0,0 +1,105 @@
+
+
+<?php if (!$player->special_mission): ?>
+
+<div class="contentDiv">
+    <h2 class="contentDivHeader">
+        Special Missions
+    </h2>
+
+    <p>
+    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nam nec est ultricies, consectetur lectus sit amet, 
+    elementum nisl. Ut euismod est imperdiet nisi finibus egestas. Fusce et turpis vitae ipsum facilisis faucibus 
+    eget in dolor. Morbi vehicula pretium nisl ut elementum. Proin elementum erat et sodales tincidunt. Etiam nec 
+    tortor pretium, pharetra nibh sit amet, semper dui. Aenean in quam in quam sagittis aliquam.
+    </p>
+    <p>
+    Vivamus ut viverra nisl. Quisque at hendrerit orci. Suspendisse volutpat augue eget velit dapibus, nec 
+    consectetur nisi suscipit. Vivamus a odio quis purus blandit luctus eget ac felis. Morbi rhoncus dapibus arcu 
+    in efficitur. Nunc luctus rhoncus nisl et vulputate. Donec ornare a libero eget venenatis. Quisque efficitur 
+    auctor tempus. Suspendisse vel consequat diam, sit amet fermentum tellus. Etiam eu nisi feugiat, sollicitudin
+    mi sed, mattis tellus. Aliquam interdum turpis vitae felis posuere pretium. Curabitur lobortis elit vel 
+    accumsan vulputate. 
+    </p>
```

Comment:
Yeah, I can try to write something up on this but I don't know if you want to have Princess try to come up with some lore on it. It would just be some background/explanation on the missions.